pkg/kclient/interface.go

```diff
@@ -162,4 +163,11 @@ type ClientInterface interface {
 
 	// ingress_routes.go
 	ListIngresses(namespace, selector string) (*v1.IngressList, error)
+
+	// CreateJobs creates a K8s job to execute task
+	CreateJobs(job batchv1.Job, namespace string) (*batchv1.Job, error)
```

Comment:
What do you think of renaming this method into `CreateJob` instead, as it creates a single Job.